### DISCUSSION TITLE
Can't open any pathway in PathVisio; classic search broken

### DISCUSSION THREAD

**Discussion by eweitz** [wikipathways/wikipathways-help#115] [General]
For a few days, opening any existing pathways in PathVisio has errored, and search in classic.wikipathways.org has been broken.

To reproduce the PathVisio bug:
1.  Open PathVisio
2. At top, click "Plugins", hover over "WikiPathways", click "Search"
3. Enter any pathway ID into "Search For", e.g. "[WP5304](https://classic.wikipathways.org/index.php/Pathway:WP5304)"
4. Note error:

> java.rmi.RemoteException: Error on line 1: Premature end of fail.; nested exception is:
> org.xml.sax.SAXParserException; lineNumber: 1; columnNumber: 1; Premature end of file.

To reproduce the classic search bug:
1.  Go to https://classic.wikipathways.org
2.  Type "cancer" into search box, press Enter on keyboard
3. Note "Error: unable to process search" in red box on [search results page](https://classic.wikipathways.org/index.php/Special:SearchPathways?query=cancer&doSearch=1)

The PathVisio issue blocks editing pathway diagrams.  The classic search bug fortunately has a workaround -- search on the new site!  Both issues had roughly the same onset time; I noticed them first on Tuesday or Wednesday.  So I assume they have the same underlying cause.

**Comment by khanspers** [comment]
Hi @eweitz , thank you for the detailed report! Both issues were indeed caused by the same underlying problem. It was fixed earlier today; search function on classic and opening/editing via PathVisio now works as expected.

**Reply by eweitz** [reply]
Thanks for the quick fix!  I confirm it works.
